### PR TITLE
removes roboto font

### LINKED ISSUE #199: Remove Roboto font

### PR DESCRIPTION
**Pull request by astewart43-gtri**
closes #199

### CODE REVIEW

File: CoEpi.xcworkspace/xcshareddata/swiftpm/Package.resolved [hunk 1]
```diff
@@ -0,0 +1,142 @@
+{
```
Comment (i-schuetz) [inline CoEpi.xcworkspace/xcshareddata/swiftpm/Package.resolved:1]: I'm not sure why these changes appear in your PRs. What Xcode version are you using?

Comment (astewart43-gtri) [reply CoEpi.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Version 11.5 (11E608c)

The files have been removed.

Comment (mellowgeek) [reply CoEpi.xcworkspace/xcshareddata/swiftpm/Package.resolved]: For what it's worth, I'm on the same version and see those files pop up every time, too.

Comment (i-schuetz) [reply CoEpi.xcworkspace/xcshareddata/swiftpm/Package.resolved]: Same version here and I don't. Will investigate.
